wandb/util.py

```diff
@@ -1426,3 +1430,31 @@ def _log_thread_stacks():
             logger.info('  File: "%s", line %d, in %s' % (filename, lineno, name))
             if line:
                 logger.info("  Line: %s" % line)
+
+
+def check_dict_contains_artifact(d):
```

Comment:
I might be missing something, but this looks unused? Consider removing.

Reply:
Darn, I think I actually forgot to implement the case where a user inserts an artifact in a dict. Which was the intended purpose of this.

Reply:
But, looking again my implementation handles it. Removing